content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Istio’s new sidecar-less ‘Ambient Mode’ reaches Beta in version 1.22"
+description: The latest release from Istio brings service mesh Layer 4 & 7 features to production readiness without sidecars.
+publishdate: 2024-05-14
+attribution: "Lin Sun (Solo.io), for the Istio Steering and Technical Oversight Committees"
+keywords: [Istio Day,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+Istio ambient service mesh was [announced in September 2022](/blog/2022/introducing-ambient-mesh/) as an experimental branch
+that introduced a new data plane mode in Istio that did not require sidecars, offering a simplified operational experience of
+Istio. After 20 months of hard work and collaboration within the Istio community, with contributions from  Solo.io, Google,
+Microsoft, Intel, Aviatrix, Huawei, IBM, Red Hat, and others, we are excited to announce that ambient mode has reached Beta in
+version 1.22! The beta release of 1.22 indicates the features of ambient mode are now stable and ready for production workloads
+with appropriate cautions.  This is a huge milestone for Istio, bringing both Layer 4 and Layer 7 mesh features to production
+readiness without sidecars.
+
+## Why ambient mode?
+
+We listened to the feedback from the community and Istio users and observed a growing need for users wanting to implement mesh
+functions for their applications and found the complexity and overhead of sidecars from service meshes hard to overcome.
+Istio’s ambient mode has been designed for simplified operations without requiring changes or restarts to your application
+alleviating the previous barriers of complexity faced by users looking to implement service mesh.
+
+Ambient mode also enables you to effortlessly enroll your applications onto Istio without sidecars. You can now simply label
+your namespace with `istio.io/dataplane-mode=ambient`, your applications in the namespace are added to the mesh and the
+communications among your application pods in ambient are secured with mTLS, without restartarting or the need for sidecars!
+
+Some challenges that Istio sidecar users have shared with us include; how Istio can break applications after sidecars are
```

Comment:
```suggestion
Some challenges that Istio sidecar users have shared with us include: how Istio can break applications after sidecars are
```

Reply:
I don't think we want to lean in to a message that says 'break applications' too hard. Yes, it can happen, but its actually quite rare. Could instead talk about how we've made some 'transport improvements' that allow to fully 'support all types of TCP' traffic but even that is probably too much. 

You talk above about application restarts, lean into the 'pain' of the there and avoid repeating here?